bin/update-property-map.js

```diff
@@ -0,0 +1,71 @@
+const path = require('path')
+const { Client } = require('pg')
+const {
+    Property,
+} = require('@condo/domains/property/utils/serverSchema')
+const { normalizePropertyMap } = require('@condo/domains/property/utils/serverSchema/helpers.js')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+const { has } = require('lodash')
+
+
+class PropertyMapUpdater {
+    async init () {
+        const resolved = path.resolve('./index.js')
+        const { distDir, keystone, apps } = require(resolved)
+        const graphqlIndex = apps.findIndex(app => app instanceof GraphQLApp)
+        await keystone.prepare({ apps: [apps[graphqlIndex]], distDir, dev: true })
+        await keystone.connect()
+        this.context = await keystone.createContext({ skipAccessControl: true })
+        this.pg = new Client(process.env.DATABASE_URL)
+        this.pg.connect()
+
+        const { rows } = await this.pg.query(' SELECT id, map, "unitsCount" FROM "Property" WHERE "map" IS NOT NULL AND "deletedAt" IS NULL ')
+        this.properties = rows
+
+        this.properties.forEach(property => {
+            if (property.map) {
+                property.map = normalizePropertyMap(property.map)
+            }
+        })
```

Comment:
This code seems to be a part of fixing, are you sure, that it should not be placed in `fix` method?

Reply:
Done